packages/dmn-js-decision-table/src/features/decision-table-properties/components/DecisionTablePropertiesComponent.js

```diff
@@ -25,7 +27,8 @@ export default class DecisionTablePropertiesComponent extends Component {
 
     return (
       <div className="decision-table-properties">
-        <div className="decision-table-name" title={ 'Decision Name: ' + name }>
```

Comment:
Is there a good reason (translation wise) why we translate the `:` with `Decision Name`?

Reply:
There exists a need to replace the colon `:` with Chinese colon `：`(\uff1a). Besides, I suppose punctuation is commonly included according to this stackexchange answer ([link](https://softwareengineering.stackexchange.com/a/374009))